### PR TITLE
ScreamingFrog.SEOSpider version 15.2

### PR DESCRIPTION
**Pull request by profcamseo**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17559)

### CODE REVIEW

**Comment by profcamseo** [comment]
amended

File: manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: ScreamingFrog.SEOSpider
+PackageVersion: 15.2
+PackageLocale: en-US
+Publisher: ScreamingFrog
```
Comment (ItzLevvie) [inline manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.locale.en-US.yaml:6]:
```suggestion
Publisher: Screaming Frog Ltd
```

File: manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: ScreamingFrog.SEOSpider
+PackageVersion: 15.2
+PackageLocale: en-US
+Publisher: ScreamingFrog
+PackageName: SEOSpider
+License: Screaming Frog Ltd
```
Comment (ItzLevvie) [inline manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.locale.en-US.yaml:8]:
```suggestion
License: Copyright (c) Screaming Frog Ltd.
```

File: manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: ScreamingFrog.SEOSpider
+PackageVersion: 15.2
+PackageLocale: en-US
+Publisher: ScreamingFrog
+PackageName: SEOSpider
```
Comment (ItzLevvie) [inline manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.locale.en-US.yaml:7]:
```suggestion
PackageName: Screaming Frog SEO Spider
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**